frontend/packages/ceph-storage-plugin/src/components/ocs-install/install-wizard/configure.tsx

```diff
@@ -76,10 +76,14 @@ export const EncryptionFormGroup: React.FC<EncryptionFormGroupProps> = ({
   }, [encryption.clusterWide, encryption.storageClass, encryptionChecked]);
 
   const toggleEncryption = (checked: boolean) => {
-    setEncryptionDispatch(ActionType.SET_ENCRYPTION, mode, dispatch, {
+    const payload = {
       ...encryption,
       clusterWide: checked,
-    });
+    }
+    if(!checked){
+      payload.advanced = false;
```

Comment:
We should also set all encryption level and both encryption and kms states to initial value when the high level encryption box is unchecked.
This is to ensure that stale states dont cause errors,
```
      payload.advanced = false;
      payload.storageClass = false;
      payload.hasHandled = false;
```
There is already one check for clearing kms state on L88, please merge the above with this one.
https://github.com/openshift/console/blob/c29d7499e8e44868bfa33d4621fa7875cc696980/frontend/packages/ceph-storage-plugin/src/components/ocs-install/install-wizard/configure.tsx#L88-L91

Reply:
ack